image/bmp.ksy

```diff
@@ -52,53 +125,175 @@ types:
         -orig-id: bfOffBits
         type: s4
         doc: Offset to actual raw pixel data of the image
-  bitmap_core_header:
-    -orig-id: BITMAPCOREHEADER
-    doc-ref: https://msdn.microsoft.com/en-us/library/dd183372.aspx
+  bitmap_info:
+    -orig-id: BITMAPINFO
+    doc-ref: https://docs.microsoft.com/cs-cz/windows/win32/api/wingdi/ns-wingdi-bitmapinfo
```

Comment:
Please reference en-us language version to keep all documentation in English (as it is in English anyway).